tools/development/mediapipe_external_objs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+for obj in $(find ${MEDIAPIPE_HOME}/bazel-bin/external/ -name '*.o' ); do
+    arr+=($obj)
```

Comment:
Even though, this script is called by meson.build which has ${MEDIAPIPE_HOEM}, but it would be better check the existence of this variable. and also no need to separate bash scripts (mediapipe_external_objs.sh & mediapipe_internal_objs.sh).  Just use bash function with argument. 
```bash
   mediapipe_objs.sh internal
   mediapipe_objs.sh external
```

Reply:
I agree. thanks for the advice.